docs/api.md

```diff
@@ -1492,6 +1497,25 @@ If there's no element matching `selector`, the method throws an error.
 
 Shortcut for [page.mainFrame().hover(selector)](#framehoverselector).
 
+#### page.interceptFileChooser()
```

Comment:
interceptNextFileChooser? I think the name to signify that it only intercepts one.

Reply:
`interceptFileChooser` is already clear enough - no need to make it more verbose.